README.md

```diff
@@ -2,160 +2,186 @@
 
 This repo provides the landing page experience for visitors to
 [landregistry.data.gov.uk](http://landregistry.data.gov.uk). The landing page
-provides links to the various open data services (Price Paid Data Explorer,
-Standard Reports and UK Housing Price Index), and hosts the qonsole app, which
-allows users to run SPARQL queries against the linked-data dataset.
+provides links to the various open data services ([Price Paid Data
+Explorer](http://landregistry.data.gov.uk/app/ppd), [Standard
+Reports](http://landregistry.data.gov.uk/app/standard-reports) and [UK Housing
+Price Index](http://landregistry.data.gov.uk/app/ukhpi)), and hosts the qonsole
+app, which allows users to run SPARQL queries against the linked-data dataset.
 
-Please see the other repositories in the [HM Land Registry Open Data](https://github.com/epimorphics/hmlr-linked-data/)
-project for more details:
+Please see the other repositories in the [HM Land Registry Open
+Data](https://github.com/epimorphics/hmlr-linked-data/) project for more
+details:
 
 - [HMLR Common Styles](https://github.com/epimorphics/lr_common_styles)
 - [PPD Explorer](https://github.com/epimorphics/ppd-explorer)
 - [Standard Reports UI](https://github.com/epimorphics/standard-reports-ui)
 - [UKHPI](https://github.com/epimorphics/ukhpi)
 - [Qonsole (Rails)](https://github.com/epimorphics/qonsole-rails)
 
-## Production Mode
+## Developer notes
```

Comment:
There is a section here call ed "Developer notes" but the contents are not just for a audience of developers, this is a bad choice of title